examples/arithmetics/src/language-server/arithmetics.langium

```diff
@@ -26,7 +26,14 @@ Addition infers Expression:
     Multiplication ({infer BinaryExpression.left=current} operator=('+' | '-') right=Multiplication)*;
 
 Multiplication infers Expression:
-    PrimaryExpression ({infer BinaryExpression.left=current} operator=('*' | '/') right=PrimaryExpression)*;
+    Potentiation ({infer BinaryExpression.left=current} operator=('*' | '/') right=Potentiation)*;
+
+Potentiation infers Expression:
+    Modulo ({infer BinaryExpression.left=current} operator='^' right=Modulo)*;
+
+Modulo infers Expression:
```

Comment:
Suggestion: This is usually called the `Remainder` operation. However, please group it with the `Multiplication` parser rule, since it has the same associativity (i.e. operator precedence).

Reply:
Wait! Isn't the ^ operator not right-associative?!

Reply:
My comment is about the `%` operator, not `^`.

Reply:
Sorry! I totally ignored that :-(. I will provide another comment with a solution